### PR TITLE
Made product card

### PR DESCRIPTION
**Pull request by mira1zu**
- [DEMO LINK](https://mira1zu.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://mira1zu.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,67 @@
...
+            69 999 грн
+        </p>
+      </div>
+
+      <div class="product-card__buy">
+        <a
+          href="/"
+          class="product-card__buy-link"
+          data-qa="hover"
+        >Купить</a>
```
Comment (MariiaNH) [inline src/index.html:63]:
```suggestion
        >
        Купить
        </a>
```

File: src/styles/blocks/product-card.css [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  font-size: 14px;
+  font-weight: 700;
+  color: #fff;
+  text-transform: uppercase;
+  text-decoration: none;
+}
+
+.product-card__buy-link:hover {
+  color: var(--blue-accent);
+}
```
Comment (MariiaNH) [inline src/styles/blocks/product-card.css:101]: Cheklist p. 6 
https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md

**Review by MariiaNH — CHANGES_REQUESTED** [review]
great work, you paid a lot attention to the systematization of styles - looks ice) Only in one place in html I propose some changes - and please pay attention to point 6 in checklist https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md

**Review by MariiaNH — APPROVED** [review]

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,69 @@
...
+      <div class="product-card__rating">
+        <div class="product-card__stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+        </div>
```
Comment (jstmpelowycz) [inline src/index.html:41]: You could add one more class name for the stars wrapper

File: src/styles/reset.css [hunk 4]
```diff
@@ -0,0 +1,3 @@
+* {
+  margin: 0;
+}
```
Comment (jstmpelowycz) [inline src/styles/reset.css:3]: Don't recommend doing this, it may lead to unexpected behavior later on

**Review by jstmpelowycz — APPROVED** [review]
Good job 🔥🔥
There is only thing to consider, check it out

**Pull request closed without merging**